src/cascadia/LocalTests_TerminalApp/ColorSchemeTests.cpp

```diff
@@ -20,15 +20,14 @@ namespace TerminalAppLocalTests
 
     class ColorSchemeTests : public JsonTestClass
     {
-        // Use a custom manifest to ensure that we can activate winrt types from
-        // our test. This property will tell taef to manually use this as the
-        // sxs manifest during this test class. It includes all the cppwinrt
-        // types we've defined, so if your test is crashing for an unknown
-        // reason, make sure it's included in that file.
-        // If you want to do anything XAML-y, you'll need to run your test in a
-        // packaged context. See TabTests.cpp for more details on that.
+        // Use a custom AppxManifest to ensure that we can activate winrt types
+        // from our test. This property will tell taef to manually use this as
+        // the AppxManifest for this test class.
+        // This does not yet work for anything XAML-y. See TabTests.cpp for more
+        // details on that.
         BEGIN_TEST_CLASS(ColorSchemeTests)
-            TEST_CLASS_PROPERTY(L"ActivationContext", L"TerminalApp.LocalTests.manifest")
+            TEST_CLASS_PROPERTY(L"RunAs", L"UAP")
+            TEST_CLASS_PROPERTY(L"UAP:AppXManifest", L"TerminalApp.LocalTests.AppxManifest.xml")

```

Comment:
Is it possible, i wonder, to set a `UAP:AppxManifest` at the _module_ level (`MODULE_PROPERTY`) so that it applies to all _classes_ marked `RunAs=UAP`?

Reply:
For the most part, what you're saying is exactly how it works. I would recommend that it be placed in a module property. I usually throw those in the "Init" file in every test module. It doesn't look like LocalTests_TerminalApp has one yet, but it's probably time that it does because there's always going to be more test configuration steps we add.

Reply:
Also if literally every test in this module is supposed to be a UAP test, you could move the RunAs declaration up there too.